openupgrade_scripts/scripts/website_sale/15.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,22 @@
+from openupgradelib import openupgrade
+
+
+def enable_price_filter_view(env):
+    for website in env["website"].search([]):
+        view = env["ir.ui.view"].search(
+            [
+                ("key", "=", "website_sale_attribute_filter_price.pricefilter"),
+                ("website_id", "=", website.id),
+            ]
+        )
+        if view:
```

Comment:
I'm thinking now... shouldn't you remove the old view?

Reply:
In my tests, when the old module is uninstalled, the old view disappears. But if, to be on the safe side, you want to deactivate it, that's fine for me.

Reply:
But here we are not uninstalling it, but merging it with `website_sale`, so yes, please do another PR unlinking them.